README.md

```diff
@@ -1101,6 +1101,7 @@ _See also: [awesome-gis](https://github.com/sshuair/awesome-gis)_
 
 - [Geo2tz](https://github.com/noandrea/geo2tz) - Get the timezone from geo coordinates (lat, lon). `MIT` `Go/Docker`
 - [GraphHopper](https://graphhopper.com/) - Fast routing library and server using OpenStreetMap. ([Source Code](https://github.com/graphhopper/graphhopper)) `Apache-2.0` `Java`
+- [H3 Viewer](https://github.com/clupasq/h3-viewer) - Leaflet-based Uber H3 cells viewer ([Demo](https://wolf-h3-viewer.glitch.me/)) `Apache-2.0` `HTML5`
```

Comment:
- The description should explain what the software does.
- Missing `.` at the end of the description